bootstrap.lua

```diff
@@ -0,0 +1,47 @@
+-- Easy installer. Bootstrapped by http://pastebin.com/p8PJVxC4
+
+local base_url, dest = select(1,...)
+dest = dest or 'computercraft-github'
+
+print("Downloading from %s":format(base_url))
+
+local FILES = {
+	'apis/dkjson',
+	'apis/github',
+	'programs/github',
+	'github'
+}
+
+local function request(url_path)
+	local request = http.get(base_url..'/'..url_path)
```

Comment:
This doesn't work as expected.  For some reason if `request` fails CC is making `request == nil`. We should mimic what we did in the `getAPI` method:

```
local request = http.get(base_url..'/'..url_path)
local success = false
local response = nil
if request then
  success = request.getResponseCode() == 200
  response = request.readAll()
  request.close()
end
return success, response
```